pkg/config/generate.go

```diff
@@ -499,10 +505,10 @@ func randomBytes(n int) ([]byte, error) {
 	return b, nil
 }
 
-func randomStorageAccountName() (string, error) {
+func randomString(lenght int) (string, error) {
```

Comment:
rollback this change and move your secret to be genereted as part of the secrets loop (L332). Also change your secret type to a byte slice.